README.md

```diff
@@ -1007,6 +1008,25 @@ The initialization function is run from the context of the HTML element.
 It should be noted that children nodes are removed from the DOM when widget instantiation occurs, and added as children
 to the widget instance.
 
+#### Meta
```

Comment:
Maybe we should talk about the mechanism here and how to implement a meta module? then we can add sections for each of the meta (Dimensions & Intersection Observers etc) as they are implemented?